packages/babel-plugin-transform-block-scoping/test/fixtures/general/for-break-continue-closure/output.js

```diff
@@ -1,21 +1,21 @@
-for (var a of [1]) {
+var _loop = function () {
```

Comment:
The old plugins sometimes injected a hoisted `_loop` function that is reused for every iteration, and other times it injected an IIFE. Not it always injects a single hoisted function.